contracts/src/chainlink/cairo/access/ownable.cairo

```diff
@@ -46,9 +46,6 @@ namespace Ownable:
     func assert_only_owner{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
         let (owner) = Ownable_owner.read()
         let (caller) = get_caller_address()
-        with_attr error_message("Ownable: Caller is the zero address"):
```

Comment:
cc @archseer we removed the zero address check.

@qperrot let's please a single test into `contracts/test/access/behavior/ownable.ts` that demonstrates that this is not possible with latest devnet.

Reply:
Do we need to remove these checks though? I'd rather keep the contract in sync with the upstream version: https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/access/ownable/library.cairo

Reply:
Ok, if you have a strong preference here.

@qperrot let's keep the `assert_not_zero`, only in `ownable` contract, and add a comment hinting that this should not be possible anymore with introduction of fees.

Then we also add a test to demonstrate it's not possible.